racket/src/racket/src/Makefile.in

```diff
@@ -300,7 +300,7 @@ vector.@LTO@: $(srcdir)/vector.c
 	$(CC) $(ALL_CFLAGS) -c $(srcdir)/vector.c -o vector.@LTO@
 
 gmp_alpha_gcc.@LTO@: $(srcdir)/gmp/gmp_alpha_gcc.s
-	$(AS) -o gmp_alpha_gcc.@LTO@ $(srcdir)/gmp/gmp_alpha_gcc.s
+	$(CC) -o gmp_alpha_gcc.@LTO@ $(srcdir)/gmp/gmp_alpha_gcc.s
```

Comment:
DId you mean to change the alpha case here, instead of arm?

My past experience was that using $(CC) to build ".s" files (and especially using it to get CPP) creates more problems than it solves, but that experience may be out-of-date.